Last-Update: 2022-02-09
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: disable test (requires root)

--- a/libcontainer/factory_linux_test.go
+++ b/libcontainer/factory_linux_test.go
@@ -60,6 +60,7 @@
 }
 
 func TestFactoryNewTmpfs(t *testing.T) {
+t.Skip("DM - skipping privileged test")
 	root := t.TempDir()
 	factory, err := New(root, TmpfsRoot)
 	if err != nil {
